ghost/i18n/locales/ar/comments.json

```diff
@@ -1,72 +1,72 @@
-    "{{amount}} more": "{{amount}}  المزيد",
-    "{{amount}} seconds ago": "منذ {{amount}}  ثواني",
-    "{{amount}} weeks ago": "منذ {{amount}}  اسابيع",
-    "{{amount}} years ago": "منذ {{amount}}  سنوات",
+    "{{amount}} characters left": "{{amount}} حرف متبقي",
+    "{{amount}} comments": "{{amount}} تعليقات",
+    "{{amount}} days ago": "منذ {{amount}} أيام",
+    "{{amount}} hrs ago": "منذ {{amount}} ساعات",
+    "{{amount}} mins ago": "منذ {{amount}} دقيقة",
+    "{{amount}} months ago": "منذ {{amount}} أشهر",
+    "{{amount}} more": "{{amount}} المزيد",
+    "{{amount}} seconds ago": "منذ {{amount}} ثواني",
+    "{{amount}} weeks ago": "منذ {{amount}} أسابيع",
+    "{{amount}} years ago": "منذ {{amount}} سنوات",
     "1 comment": "تعليق واحد",
-    "Add comment": "اضف تعليق",
-    "Add context to your comment, share your name and expertise to foster a healthy discussion.": "قم بإضافة سياق لتعليقك، شارك اسمك و مجال خبرتك لكي نرعى محادثة مثمرة",
-    "Add reply": "ردّ",
-    "Already a member?": "هل انت مسجل بالفعل؟",
+    "Add comment": "إضافة تعليق",
+    "Add context to your comment, share your name and expertise to foster a healthy discussion.": "أضف سياقًا لتعليقك، شارك اسمك وخبرتك لتسهيل النقاش البنّاء.",
+    "Add reply": "إضافة رد",
+    "Already a member?": "هل أنت عضو بالفعل؟",
     "Anonymous": "مجهول",
-    "Become a member of {{publication}} to start commenting.": "يجب أن تصبح عضوًا فى {{publication}}  لتستطيع التعليق",
-    "Become a paid member of {{publication}} to start commenting.": "يجب ان تكون عضويتك مدفوعة فى {{publication}} لكي تستطيع التعليق",
+    "Become a member of {{publication}} to start commenting.": "اصبح عضوًا في {{publication}} لبدء التعليق.",
+    "Become a paid member of {{publication}} to start commenting.": "اصبح عضوًا مدفوعًا في {{publication}} لبدء التعليق.",
     "Cancel": "إلغاء",
     "Comment": "تعليق",
-    "Complete your profile": "إكمال ملفك الشخصى",
+    "Complete your profile": "أكمل ملفك الشخصي",
     "Delete": "حذف",
-    "Deleted member": "الاعضاء المحذوفين",
-    "Discussion": "النقاش",
+    "Deleted member": "عضو محذوف",
+    "Discussion": "نقاش",
     "Edit": "تعديل",
-    "Edit this comment": "عدّل هذا التعليق",
+    "Edit this comment": "تعديل هذا التعليق",
     "edited": "تم التعديل",
-    "Enter your name": "اكتب اسمك",
-    "Expertise": "مجال تخصصك",
-    "Founder @ Acme Inc": "مؤسس لدى شركة أكمي",
-    "Full-time parent": "أب بدوام كامل",
-    "Head of Marketing at Acme, Inc": "رئيس وحدة التسويق لدى شركة أكمى",
-    "Hide": "اخفاء",
-    "Hide comment": "اخف التعليق",
-    "Jamie Larson": "فلان الفلانى",
-    "Join the discussion": "انضم للمحادثة",
+    "Enter your name": "أدخل اسمك",
+    "Expertise": "مجال التخصص",
+    "Founder @ Acme Inc": "مؤسس @ شركة أكمي",
+    "Full-time parent": "والد بدوام كامل",
+    "Head of Marketing at Acme, Inc": "رئيس التسويق في شركة أكمي",
+    "Hide": "إخفاء",
+    "Hide comment": "إخفاء التعليق",
+    "Jamie Larson": "جيمي لارسون",
```

Comment:
the original translation for "Jamie Larson" is better, as it is intended to be a generic name, not a literal transliteration.